docs/docs/graphql.md

```diff
@@ -1490,7 +1490,14 @@ If your project uses [Docusaurus](https://docusaurus.io), the generated commente
 
 The following is some basic setup information, but please consult [Docusaurus](https://docusaurus.io) and the [graphql-markdown](https://graphql-markdown.github.io) for latest instructions.
 
-1. Add `docs` to your `workspaces` in the project's `package.json`:
+1. Install Docusaurus (if you have not done so already)
```

Comment:
This is repeated at step 3. Also, see note below about docs folder being created when running this command.

Reply:
Does this mean I should remove the step from the instructions?

Reply:
No, leave this here. We should remove step three, but it will cause renumbering for everything else. I will try this with suggestions.

Reply:
removed step 3 and renumbered.